integration_tests/tests/consistency/consistency_columns.sql

```diff
@@ -0,0 +1,52 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+/* This test is to make sure the final columns produced are the same between versions.
+Only one test is needed since it will fetch all tables and all columns in each schema.
+!!! THIS TEST IS WRITTEN FOR BIGQUERY!!! */
+{% if target.type == 'bigquery' %}
+with prod as (
+    select
+        table_name,
+        column_name,
+        /* Need the case since the prod version is uncasted, and bigquery automatically produces a value of "bignumeric" or "float"
+        while the dev is casted as dbt.type_numeric() and produces a value of "numeric" */
+        case when lower(data_type) like '%numeric%' or lower(data_type) like '%float%' then 'numeric'
+            else data_type 
+            end as data_type
```

Comment:
Now that we are making this a breaking change, would it make more sense to remove this case when statement? We know it will fail for this release (why it is breaking now), but it should not need this case statement for future updates and could risk us not catching datatype mismatches.